sdk/translation/azure-ai-translation-text/README.md

```diff
@@ -37,10 +37,12 @@ pip install azure-ai-translation-text
 
 #### Create a Translator service resource
 
-You can create Translator resource following [Create a Translator resource][translator_resource_create].
+You can create Translator resource following [Create a Translator resource][translator_resource_create].  For general information about authentication with a cognitive services resource please refer to the documentation [here][cognitive_auth].
```

Comment:
`[cognitive_auth]` - again, this might be an issue as this is related to auth only for custom endpoint. Our MT endpoints have different auth methods.

Reply:
Removed, this was too vague.